lib/root.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Router, Route } from 'react-router'
+import BrowserHistory from 'react-router/lib/BrowserHistory'
+import { Provider } from 'redux/react'
+import { createDispatcher, createRedux, composeStores } from 'redux'
+import { loggerMiddleware, thunkMiddleware } from './middleware'
+import { About, Application, Github, GithubRepo,
+  GithubUser } from './components'
+import * as stores from './stores'
+
+export default class Root extends React.Component {
+    constructor(props) {
+      super(props)
+      this.history = new BrowserHistory()
+    }
+
+    render() {
+      const dispatcher = createDispatcher(
```

Comment:
This should be done outside `render`. It's fine to place this before the `Root` definition.